builders/config.json.example

```diff
@@ -1,3 +1,16 @@
+# It's critical to correctly configure the credentials
+# here or the building process will fail with permission
+# errors.
+#
+# For registry.svc.ci.openshift.org, the "auth" is 
+# equal to the token that is given when getting
```

Comment:
IIRC the token can't be used directly in the auth file, you first need to do something like:
```
podman login --authfile <authfile> -u <username> -p <token> https://registry.svc.ci.openshift.org
```

Reply:
Interesting comment, I'll re-try again and update the doc if needed.

Reply:
@mandre I updated it, let me know if we can resolve this conversation.